packages/tokens/src/figma-tokens/_docs.json

```diff
@@ -773,6 +763,54 @@
         "value": "Visually Hidden",
         "type": "text"
       }
+    },
+    "radio": {
+      "default": {
+        "title": {
+          "value": "Radio",
+          "type": "text"
+        },
+        "description": {
+          "value": "Radios allow the user to select a single option from a group. \nRadios are designed to be used with radio groups.",
```

Comment:
Does that mean we need a `<syn-radio-group />` element to make use of `<syn-radio />`? In default html, radios are defined as a group of radio controls that have the same `name` attribute and do not require a wrapper.

Reply:
Yes. https://shoelace.style/components/radio-group How did you solve this with DaVinci?

Reply:
In our current library, there is just the plain element without wrapper and without label. As we do not offer real form support, this is not that big of a deal currently, but we want to make it better ;)